javavm/graalvm_tcp_server/app/src/main/java/smartcontract/SmartScalarMult.java

```diff
@@ -0,0 +1,24 @@
+package smartcontract;
+
+public class SmartScalarMult implements SmartContract {
+
+  public byte[] Execute(byte[] input) throws SmartContractException {
+    final double ITERATIONS = 1e6;
+
+    if (input.length != crypto_core_ed25519_SCALARBYTES) {
+      throw new SmartContractException("failed to process: input should be 8 bytes long!");
+    }
+
+    byte[] output = new byte[crypto_core_ed25519_SCALARBYTES];
+
+    SmartCrypto sc = new SmartCrypto();
+
+    for (double i = 0; i < ITERATIONS; i++) {
+      sc.scalarmult_ed25519_base_noclamp(output, input);
+    }
+
+    return output;
+  }
+
+  public final int crypto_core_ed25519_SCALARBYTES = 32;
```

Comment:
a very small detail, but the good practice (at least 4 years ago 😁 ) was to put global variables and attributes at the top of the class.

Reply:
Some automagic tool pushed that down the file. I didn't mind as we are dealing with a one small page class. But I'll keep the good practice. Thx.